src/host/screenInfo.cpp

```diff
@@ -2084,8 +2058,6 @@ void SCREEN_INFORMATION::SetPopupAttributes(const TextAttribute& popupAttributes
 void SCREEN_INFORMATION::SetDefaultAttributes(const TextAttribute& attributes,

```

Comment:
does this need some sort of VT translation?

Reply:
The old code didn't translate this AFAIK, so the new one isn't doing it either atm.